src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointClusterer.java

```diff
@@ -63,12 +62,35 @@ private void buildTree( final Iterator<BreakpointEvidence> evidenceItr ) {
 
     private boolean hasEnoughOverlappers( final SVInterval interval ) {
         final Iterator<SVIntervalTree.Entry<List<BreakpointEvidence>>> itr = evidenceTree.overlappers(interval);
```

Comment:
It seemed a little redundant to add a whole new test to the new `BreakpointClustererUnitTest`, so I just added a bunch of assertions about this method to the test methods there.